bin/check-container-logs.rb

```diff
@@ -83,33 +80,57 @@ class ContainerLogChecker < Sensu::Plugin::Check::CLI
          long: '--seconds-ago SECONDS',
          required: false
 
+  option :check_all,
+         description: 'Check all containers. Only running containers are checked by default',
+         short: '-a',
+         long: '--all',
+         default: false,
+         boolean: true
+
+  option :disable_stdout,
+         description: 'Disable the check on stdout logs',
+         short: '-1',
+         long: '--no-stdout',
+         default: true,
+         boolean: true,
+         proc: proc { false }
```

Comment:
what does this `proc` do?

Reply:
I think the attempt was to negate whatever was passed? I don't think that actually works though.

Reply:
If I am correct we should just name it `--stdout` and have it default to false.

Reply:
The purpose of 'proc' was indeed to reverse the default false / true behavior.
Indeed, I didn't provid a test artifact but should to show you i tested it :).
The actual version check stdout and stderr by default and i didn't want to break this comportement, so I had come to this solution to filter which output on we would like to filter.
But now that you point me on this, maybe you would like to see it implemented like this ?
- Default if --stdout and --stderr are not provided : show both
- if --stdout : show only stdout
- if --stdout : show only stderr
- if --stdout and --stderr are both provided : show both

I could implement this workflow if you liked to, but i think it will be less readable.

Reply:
I am ok with that, though I think we should put a comment above it to explain it so when we come back in 6 months we are not scratching our heads trying to figure it out.